app/controllers/v0/institutions_controller.rb

```diff
@@ -30,10 +30,7 @@ def index
       }
 
       if @query.key?(:fuzzy_search)
-        order_by = <<-SQL
-          (SIMILARITY(institution, :search_term) + SIMILARITY(city, :search_term)) / 2 DESC,
-          institution ASC
-        SQL
+        order_by = 'SIMILARITY(institution, :search_term) * COALESCE(gibill, 0) DESC, institution'
```

Comment:
Just curious as to why the SIMILARITY on city is no longer needed

Reply:
Longer answer: This is getting in front of users for testing and that's driving us to try different things until they think the "feel" is right.

Short answer: Tried it, didn't like it.

Reply:
👍